src/core/reference/include/ngraph/runtime/reference/scatter_nd_update.hpp

```diff
@@ -35,10 +35,22 @@ void scatterNdUpdate(const dataType* const inputData,
         return padding;
     }();
 
+    std::vector<indicesType> indicesCopy(indices, indices + shape_size(indicesShape));
```

Comment:
Why do you need a copy of indices?
I don't see that indices are changing...

Reply:
hi @ilyachur , here I need to change negative indices to positive.
I have to make copy, because I can't update originial value of indices, it will araise crash.
https://github.com/openvinotoolkit/openvino/pull/17219/files/ffc8fed0b0b4e80ff0a0d6243aa74d6f63860376#diff-7c7092e635ee31af64501f9e1aaef6ddd49adafe3f40669c2288428be0835ebbR49 **Line49** will update value of indice, it pass pointer, from `indicesCopy` to `coord`